### PR TITLE
initial change of usage and description

### LINKED ISSUE #124: Add usage and descriptions to each Cobra Command

### PR DESCRIPTION
**Pull request by 6missedcalls**
Refactor of the CLI command descriptions and usage. 
## Changes

- ```speedway object get``` command changed to ```speedway object query```

### New Features *(required)*
N/A

### Deprecations *(required)*
N/A

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes #124 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Review by ntindle — APPROVED** [review]

File: cmd/speedway-cli/speedwaycmd/object/object.go [hunk 1]
```diff
@@ -9,12 +9,11 @@ import (
...
-		Use:   "object",
-		Short: "Provides commands for managing objects on the Sonr Network",
-		Long: `Build and Get objects on the Sonr Network. 
-		Build an object for an existing schema and upload it to the network.
-		Get an existing object from the network.
-		For more information, visit https://docs.sonr.io/.`,
+		Use:   "object [subcommand]",
+		Short: "Build and query objects on the Sonr Network.",
+		Long: `Build and query objects on the Sonr Network.
+		You can create a new object with the 'build' subcommand.
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/object/object.go:15]: you can remove the "you can"

File: cmd/speedway-cli/speedwaycmd/object/object.go [hunk 2]
```diff
@@ -9,12 +9,11 @@ import (
...
+		You can query an existing object with the 'query' subcommand.`,
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/object/object.go:16]: can remove "you can"

File: cmd/speedway-cli/speedwaycmd/registry/Login.go [hunk 3]
```diff
@@ -16,8 +16,10 @@ import (
 func bootstrapLoginCommand(ctx context.Context, logger *golog.Logger) (loginCmd *cobra.Command) {
 	loginCmd = &cobra.Command{
 		Use:   "login",
-		Short: "Use: speedway registry login",
-
+		Short: "Login to an existing account on the Sonr Network.",
+		Long: `Login to an existing account on the Sonr Network.
+		You will be prompted to enter your password to decrypt your vault.
+		You may be also be prompted to enter your computer password to allow Speedway to access a keypair for you.`,
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/registry/Login.go:22]: you can remove "you will" and "you may"

File: cmd/speedway-cli/speedwaycmd/registry/Registry.go [hunk 4]
```diff
@@ -9,9 +9,13 @@ import (
 
 func BootstrapRegistryCommand(ctx context.Context, logger *golog.Logger) (registryCmd *cobra.Command) {
 	registryCmd = &cobra.Command{
-		Use:   "account",
-		Short: "Provides commands for managing your account on the Sonr Network",
-		Run:   func(cmd *cobra.Command, args []string) {},
+		Use:   "account [subcommand]",
+		Short: "Provides commands for managing your account on the Sonr Network.",
+		Long: `Provides commands for managing your account on the Sonr Network.
+		You can create a new account or login to an existing account.
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/registry/Registry.go:15]: you can remove "you can"

File: cmd/speedway-cli/speedwaycmd/registry/register.go [hunk 5]
```diff
@@ -19,8 +19,10 @@ import (
 func bootstrapCreateAccountCommand(ctx context.Context, logger *golog.Logger) (createCmd *cobra.Command) {
 	createCmd = &cobra.Command{
 		Use:   "create",
-		Short: "Use: registry create",
-
+		Short: "Creates a new account on the Sonr Network.",
+		Long: `Creates a new account on the Sonr Network.
+		You will be prompted to enter a password to encrypt your vault.
+		You will also be prompted to enter your computer password to allow Speedway to create a new keypair for you. (Please do not delete this keypair from your keychain.)`,
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/registry/register.go:25]: you can remove "You will" and "You can"

File: cmd/speedway-cli/speedwaycmd/schema/schema.go [hunk 6]
```diff
@@ -9,9 +9,12 @@ import (
 
 func BootstrapSchemaCommand(ctx context.Context, logger *golog.Logger) (schemaCmd *cobra.Command) {
 	schemaCmd = &cobra.Command{
-		Use:   "schema",
+		Use:   "schema [subcommand]",
 		Short: "Provides commands for managing schemas on the Sonr Network",
-		Run:   func(cmd *cobra.Command, args []string) {},
+		Long: `Provides commands for managing schemas on the Sonr Network.
+		You can create a new schema with the 'build' subcommand.
+		You can query an existing schema with the 'query' subcommand.`,
```
Comment (joshLong145) [inline cmd/speedway-cli/speedwaycmd/schema/schema.go:16]: you can remove "you can"

**Review by joshLong145 — CHANGES_REQUESTED** [review]

**Review by mcjcloud — APPROVED** [review]

**Pull request merged**